build.gradle

```diff
@@ -2,23 +2,27 @@
 
 buildscript {
     repositories {
+        google()
         jcenter()
         maven {
             url 'https://maven.google.com/'
             name 'Google'
         }
+        maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.2.0'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.0'
         classpath 'com.github.dcendents:android-maven-gradle-plugin:2.0'
+        classpath 'io.fabric.tools:gradle:1.25.4'
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
 }
 
 allprojects {
     repositories {
+        google()
         jcenter()
         maven {
```

Comment:
REmove this also

Reply:
I think [this line](https://github.com/onaio/kujaku/blob/bec90b958819129b10fb4ac6d552605f8cae12cd/build.gradle#L27) is a duplicate of `google()`

Remove it, everything should work ok

Reply:
This change does not work on my machine.

Reply:
Again, we discussed this.

Reply:
If there is another solution, please share it.